scripts/OverlaySurvey.py

```diff
@@ -285,6 +323,20 @@ def main():
                                 help="output file for the augmented graph")
     parser_augment.set_defaults(func=augment)
 
+    parser_simplify = subparsers.add_parser('adjustForSimulation',
+                                            help="adjust the master graph"
+                                            "for simulation purposes")
+    parser_simplify.add_argument("-gmli",
+                                 "--graphmlInput",
+                                 required=True,
+                                 help="input master graph"
+                                 "(i.e., output of the augment command)")
+    parser_simplify.add_argument("-json",
```

Comment:
This command is not right: it actually performs 2 distinct things.

The paradigm in this script is to have various "transforms" that operate on graph data, and removing nodes or connections would be such a transform that would be run 

* It simplifies the graph, by removing nodes based on some criteria.
* It converts a graph into json.

I'm pretty sure we need that second one  (we can call it `flatten`), that takes a graph and "flattens" it in JSON form (undirected).

Reply:
Replaced the command with the `flatten` command.